src/Turbo/Resources/assets/src/turbo_controller.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Controller } from 'stimulus';
+import * as Turbo from '@hotwired/turbo';
```

Comment:
Does importing it actually *enable* TurboDrive? Or does that require more work?

Reply:
Yes it does (tested locally :) ).